p2panda-rs/src/operation/operation_fields.rs

```diff
@@ -57,6 +57,9 @@ impl Validate for OperationValue {
     }
 }
 
+/// A `RelationList` can be used to reference multiple foreign documents from a document field.
+pub type RelationList = Vec<Relation>;
```

Comment:
Nice to have an own definition for `RelationList` I think i already needed this in a few other places.